src/main/resources/db/migration/V202408271754__test.sql

```diff
@@ -1,5 +1,4 @@
-create table test_me
-(
+create table test_me(
```

Comment:
remove all changes from this migration
it should be like this
```
create table test_me
(
```

Reply:
Changed